docs/core/components/viewer.component.md

```diff
@@ -186,10 +186,9 @@ npm install pdfjs-dist
 
 ```ts
 // PDF.js
-require('pdfjs-dist/web/compatibility.js');
 const pdfjsLib = require('pdfjs-dist');
-pdfjsLib.PDFJS.workerSrc = './pdf.worker.js';
-require('pdfjs-dist/web/pdf_viewer.js');
+pdfjsLib.PDFJS.workerSrc = './pdf.worker.min.mjs';
+require('pdfjs-dist/legacy/web/pdf_viewer.mjs');
```

Comment:
this points to legacy, but below it copies the regular one with copywebpack plugin